content_handler/framebuffer_skia.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_CONTENT_HANDLER_FRAMEBUFFER_SKIA_H_
+#define FLUTTER_CONTENT_HANDLER_FRAMEBUFFER_SKIA_H_
+
+#include "lib/ftl/macros.h"
+#include "mojo/services/framebuffer/interfaces/framebuffer.mojom.h"
+#include "third_party/skia/include/core/SkSurface.h"
+
+namespace flutter_content_handler {
+
+class FramebufferSkia {
+ public:
+  FramebufferSkia();
+  ~FramebufferSkia();
+
+  void Bind(mojo::InterfaceHandle<mojo::Framebuffer> framebuffer,
+            mojo::FramebufferInfoPtr info);
+
+  mojo::Framebuffer* get() const { return framebuffer_.get(); }
+  const sk_sp<SkSurface>& surface() { return surface_; }
```

Comment:
Yeah, I'll try for MakeRasterDirectReleaseProc to make the surface own its backing store.  Even if I convert this to an `SkSurface*`, that won't stop someone from making an `sk_sp` out of it.